### ISSUE TITLE
Test matchers are `equal`, potential for false positives

### ISSUE THREAD

**Issue by DoctorHowser** [PrimeAcademy/assignment-unit-2.1#15]
Tests want `Try again later.` vs  failing with `Try again later`
`expect(diceRoll).to.equal('Try again later.');` vs
`expect(diceRoll).to.equal(/Try again later/i);`

This may require some adjustment?

**Comment by christopher-black** [comment]
I agree. These tests should be more flexible and pass even if the punctuation is missing.
